.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# All of the code in this repo is owned by the Security Experience team.
+* @AzureAD/security-experience
```

Comment:
```suggestion
* @AzureAD/security-experience

```
I guess add a newline at the end of file could be better.

Reply:
Good suggestion, thanks! 😎